src/main/scala/dsptools/Chisel3TestersCompatibility.scala

```diff
@@ -0,0 +1,14 @@
+package chisel3.iotesters
```

Comment:
Well, I'd argue that currently, things are quite brittle and poor for extensibility. 

If you look, flatten is only functioning on Aggregates. It's just used to get the underlying elements of Aggregates so that you can peek at them one by one. It's necessary, b/c that's how the original PeekPokeTester implemented things, and I needed to change the function that was being called on the individual elements. An alternative would be to completely rewrite a function (that just recursively goes through elements), but that's what flatten gives you, so doing so seems incredibly redundant. 

IMO, API can be hidden (i.e. not in some kind of user doc), but still usable by "advanced users". If not, I'm literally going to take your function, copy and paste it somewhere, and remove "private" -- and you wonder why things get forked. 

When you ask for uses cases, to me, that implies more personal reasons for why I want to do X, Y, Z. This isn't even at that level of "use case." All I'm trying to do is change some small details in the way the original PeekPokeTester works that's more suitable for more informative testing of DSP without messing around with the original PeekPokeTester/DspTester (b/c doing that would make people even more unhappy, I presume). These functions were used in the original PeekPokeTester itself. If I want to change a tiny thing in my extended version of PeekPokeTester while keeping original functionality, I require a lot of copy/pasting of functions that are otherwise "private" to anything outside of PeekPokeTester. 

Re: bigIntToStr: Ask the person who first decided to use it in PeekPokeTester (or maybe it was VerilatorBackend). It's not always that straightforward for someone to realize that a function already exists unless they've seen it before... 

Anyways, can I expect a change to support these within the next n days, or should I just merge this into dsptools the way it currently is, since it seems like it's not as simple as just publicizing stuff? @chick Unless you have a better solution... 

---

Also, re: addPin, if I'm not the only one asking to keep the function in, then IMO, saying it's "unsafe" is not a good justification for taking it away.  The average person won't even see/know the function exists, but super users have reasons for using them, and it just serves to alienate them if you do remove such functionality without an acceptable backup in place. [If nothing else, it served as a workaround for features that didn't exist in Chisel...].